_g3/template-parts/entry.php

```diff
@@ -98,8 +98,9 @@
 							<dd class="tagcloud"><?php echo wp_kses_post( $tags_list ); ?></dd>
 							</dl>
 						</div><!-- [ /.entry-tag ] -->
-					<?php } ?>
-
+					<?php }
+					echo wp_kses_post( apply_filters( 'lightning_post_taxonomies_after', '' ) ); 
+					?>
```

Comment:
@drill-lancer 空のフィルター使ってるけどアクションフックでだめな理由ってあります？
他の箇所もアクションフックで書いてるからアクションの方が統一性もあるし挿入だけでも使いやすいかなと。
```suggestion
					<?php }
					add_action( 'lightning_entry_footer_append' ); 
					?>
```

Reply:
vk-posts の方がフィルターだから同じ処理が使えるようにフィルターにしたのかな？
とは言え、そうだとしたら、外部要因の都合で元々の記述の統一性を崩すのはよくないので add_action の方が良いのではないかと思いますがどうでしょう？

Reply:
> vk-posts の方がフィルターだから同じ処理が使えるようにフィルターにしたのかな？

まさにその通りでこのほうがコードを書く行が少なくて済むなのでそうしました。

Reply:
@drill-lancer あー、やはりそうですかー。

* フィルターフックはそもそも既に存在する値の改変のためにあるもので、処理の追加が主目的ではない。
* Lightning で同様に処理の追加用のフックは全て do_action を利用している。WordPress コアも追加だけの処理は全て do_action
* vk posts の方も元々は do_action にしたかったが、即表示しない get_ にする必要があったため、vk-posts はフィルターフックしか使えないという事情がるのでやむなくフィルターになっているだけ

という理由で Lightning 側では do_action で追加にしました (￣人￣)